samples/cli/nomsconfig/README.md

```diff
@@ -1,54 +1,60 @@
 
-This directory contains an [example](.gitconfig) config to try out. If you lke what 
-you see once you've kicked the tires, you can create your own *.nomsconfig*'s wherever 
-you need one.
+# Example
+
+This example defines a simple [.nomsconfig](.nomsconfig) to try:
 
-A *.gitconfig* file can contain a single default database definition and 
-any number of database aliases. Here's are the definitons from the example:
- 
 ```
 # Default database URL to be used whenever a database is not explictly provided
-[default]
+[db.default]
 url = "ldb:.noms/tour"
 
 # DB alias named `origin` that refers to the remote cli-tour db 
 [db.origin]
 url = "http://demo.noms.io/cli-tour"
 
-# DB alias named `tour` provides an explicit name for the default database
-[db.tour]
-url = "ldb:.noms/tour"
+# DB alias named `temp` that refers to a noms db stored under /tmp 
+[db.temp]
+url = "ldb:/tmp/noms/shared
+
 ```
 
-The *[default]* definition will be used implicitly whenever a database is
-required but not specified. 
+The *[db.default]* section:
 
-The *origin* and *tour* aliases can be used in place of a dataspec.  
+ - Defines a default database
+ - It will be used implicitly whenever a database url is ommitted in a command
+ - It will be used implicitly whenever a database url is ommitted in a command
 
-You can try out this config by running the noms cli in this directory. Here are some 
-commands to try and what to expect:
+The *[db.origin]* and *[db.shared]* sections
+
+ - Define aliases that can be used wherever a db url is required
+ - You can define additional aliases by adding *[db.**alias**]* sections using any **alias** you prefer
+
+You can kick the tires by running an noms commmand from this directory. Here are some examples and what to expect:
 
 ```
-noms ds          # noms ds ldb:.noms/tour 
-noms ds origin   # noms ds http://demo.noms.io/cli-tour
+noms ds          # -> noms ds ldb:.noms/tour
+noms ds default  # -> noms ds ldb:.noms/tour
+noms ds origin   # -> noms ds http://demo.noms.io/cli-tour
 
-noms sync origin::sf-film-locations sf-films   # sync from remote to local
+noms sync origin::sf-film-locations sf-films   # sync ds from origin to default
 
-noms log sf-films                    # noms log ldb:.noms/tour::sf-films
-noms log origin::sf-film-locations   # noms log http://demo.noms.io/cli-tour::sf-film-locations
+noms log sf-films                    # -> noms log ldb:.noms/tour::sf-films
+noms log origin::sf-film-locations   # -> noms log http://demo.noms.io/cli-tour::sf-film-locations
 
-noms show '#1a2aj8svslsu7g8hplsva6oq6iq3ib6c'         # noms show ldb:.noms/tour::'...'
-noms show origin::'#1a2aj8svslsu7g8hplsva6oq6iq3ib6c' # noms show http://demo.noms.io/cli-tour::'...'
+noms show '#1a2aj8svslsu7g8hplsva6oq6iq3ib6c'         # -> noms show ldb:.noms/tour::'...'
+noms show origin::'#1a2aj8svslsu7g8hplsva6oq6iq3ib6c' # -> noms show http://demo.noms.io/cli-tour::'...'
```

Comment:
I'm not sure what the '...' is in these comments. I not certain whether it's a notation your using to avoid retyping the hash or if it's legitimate syntax that I can use on the command line.